mitochondria-pipeline.inputs.json

```diff
@@ -0,0 +1,59 @@
+{
+  "##_COMMENT1": "Inputs",
+  "MitochondriaPipeline.wgs_aligned_input_bam": "gs://gatk-best-practices/mitochondria-pipeline/G97753.NA12878.bam",
+  
+  "##_COMMENT2": "Reference and Resources",
+  "#MitochondriaPipeline.ref_fasta": "File? (optional)",
```

Comment:
It seems confusing to have `#` at the beginning of some of these inputs because if you wanted to actually use those values they'd have to remember to delete the `#`.  Would it make sense to have optional inputs in a separate json file?

Reply:
There hasn't been any issues about this previously and I'd rather avoid having additional files. 
We could instead remove the optional variables, to make sure users don't accidently add their input without removing the comment.

Reply:
I don't feel that strongly. If this is the way you've been doing it and there haven't been any issues, that's fine with me.